reactnative/reactnative-todo/src/components/journey/journey-handler.js

```diff
@@ -0,0 +1,233 @@
+/*
+ * forgerock-react-native-sample
+ *
+ * journey-hook.js
+ *
+ * Copyright (c) 2021 ForgeRock. All rights reserved.
+ * This software may be modified and distributed under the terms
+ * of the MIT license. See the LICENSE file for details.
+ */
+
+import { FRStep } from '@forgerock/javascript-sdk';
```

Comment:
Why are we using JS SDK here?

Reply:
ha good question we use FRStep from javascript-sdk , which has some parsing logic..i have to look this deep and see if we can remove..but again not in this story

import { FRStep } from '@forgerock/javascript-sdk';

 const step = new FRStep(next);